test/main/accounts/index.test.js

```diff
@@ -77,7 +78,9 @@ beforeEach(done => {
 
   Accounts.add(account2.address)
   Accounts.add(account.address, account, (err, account) => {
-    Accounts.setSigner(account.address, done)
+    if (account) {
```

Comment:
can this ever be undefined? looks like it will always return an account unless we don't pass an address

Reply:
Think it was a hangover from trying things out